### DISCUSSION TITLE
How to use recipes

### DISCUSSION THREAD

**Discussion by campaignupgrade** [deployphp/deployer#2867] [General]
I think recipes are meant to be copied into the project, replacing the default deploy.php created by `dep init`, yes?

If so I'd be happy to update the documentation because it's not clear how recipies like "wordpress" are meant to be used.

**Comment by pokgak** [comment]
AFAIK recipes gives you a set of command that can be used with a preset defaults, no need to be copied into your `deploy.php` file. Example when using the Laravel recipe, I just need 'import' it at the top of the file then it will override the default `deploy` steps with deploy steps declared in the Laravel recipe. So when you do a `dep deploy` it will deploy it how a Laravel application should be deployed.
